ci/settings.yml

```diff
@@ -377,6 +381,269 @@ jobs:
+    plan:
+    - in_parallel:
+      - { get: git }
+      - { get: bosh-cli, trigger: true }
+    - task: update-bosh-cli
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: (( grab meta.image.name ))
+            tag:        (( grab meta.image.tag ))
+        inputs:
+        - name: git
+        - name: bosh-cli
+        outputs:
+        - name: pushme
+        run:
+          path: ./git/ci/scripts/update-blob
+        params:
+          REPO_ROOT:        git
+          REPO_OUT:         pushme
+          BLOB_DIR:         bosh-cli
+          BLOB_NAME:        bosh-cli
+          BLOB_BINARY:      bosh-cli-*-linux-amd64
+          BLOB_DESTINATION: jumpbox/bins/bosh
+          BLOB_CLEANUP:     jumpbox/bins/bosh
+          AWS_ACCESS_KEY:   (( grab meta.aws.access_key ))
+          AWS_SECRET_KEY:   (( grab meta.aws.secret_key ))
+          BRANCH:           (( grab meta.github.branch ))
+      on_success:
+        put: notify
+        params:
+          topic:   (( concat meta.shout.topic "-$BUILD_JOB_NAME" ))
+          message: blob ingestion job '$BUILD_JOB_NAME' succeeded.
+          ok:      yes
+          link:    (( grab meta.shout.links.build ))
+      on_failure:
+        put: notify
+        params:
+          topic:   (( concat meta.shout.topic "-$BUILD_JOB_NAME" ))
+          message: blob ingestion job '$BUILD_JOB_NAME' failed.
+          ok:      no
+          link:    (( grab meta.shout.links.build ))
+    - put: git
+      params:
+        rebase: true
+        repository: pushme/git
+      on_success:
+        put: notify
+        params:
+          topic:   (( concat meta.shout.topic "-$BUILD_JOB_NAME" ))
+          message: blob ingestion job '$BUILD_JOB_NAME' succeeded.
+          ok:      yes
+          link:    (( grab meta.shout.links.build ))
+      on_failure:
+        put: notify
+        params:
+          topic:   (( concat meta.shout.topic "-$BUILD_JOB_NAME" ))
+          message: blob ingestion job '$BUILD_JOB_NAME' failed.
+          ok:      no
+          link:    (( grab meta.shout.links.build ))
+   # }}}
+  - name: credhub-cli # {{{
+    public: true
+    plan:
+    - in_parallel:
+      - { get: git }
+      - { get: credhub-cli, trigger: true }
+    - task: update-credhub-cli
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: (( grab meta.image.name ))
+            tag:        (( grab meta.image.tag ))
+        inputs:
+        - name: git
+        - name: credhub-cli
+        outputs:
+        - name: pushme
+        run:
+          path: ./git/ci/scripts/update-blob
+        params:
+          REPO_ROOT:        git
+          REPO_OUT:         pushme
+          RELEASE_DIR:      credhub-cli
+          BLOB_SOURCE:      https://github.com/cloudfoundry-incubator/credhub-cli/releases/download/%s/credhub-linux-%s.tgz
```

Comment:
this won't work as printf will only replace the first `%s` with the version
we probably need to revise the update-blob script

Reply:
TIL that `printf` in bash is pretty useless and not really powerful.

How would you solve it? 

The default *Substring Replacement* should work, no?

```bash
BLOB_SRC_URL="${BLOB_SOURCE//%s/$VERSION}";
```

If so we could even consider giving this parameter and the substitution a more descriptive placeholder name?

E.g.:

```bash
#!/bin/bash 
VERSION="0.8.15";
BLOB_SOURCE="https://github.com/cloudfoundry-incubator/credhub-cli/releases/download/{version}/credhub-linux-{version}.tgz";
BLOB_SRC_URL="${BLOB_SOURCE//\{version\}/$VERSION}";
```

Reply:
i have revised the update-blob script 
https://github.com/cloudfoundry-community/jumpbox-boshrelease/commit/32955d3c6d79e693bd70aba483d94ea335235103

Reply:
@ramonskie but the changes you made are related to untar tgz archives, right? It is not related to the issue that printf will just replace the first `%s` in the url?

Reply:
we rely on the concourse getting the right version with the getting globs..
so concourse checks if there is a new version.
this will store the globs like for fly
```
params:
       globs:
        - fly-*-linux-amd64.tgz
```
and the update-blob script will handle the unpack and placement

Reply:
Okay, so I removed `BLOB_SOURCE` for credhub and fly and added the globs param. That should solve the issue, right?